### PR TITLE
Fix api-server flags in example

### PR DESCRIPTION
**Pull request by ecnahc515**

### CODE REVIEW

**Comment by ecnahc515** [comment]
cc @aaronlevy

File: .gitignore-e [hunk 1]
```diff
@@ -0,0 +1 @@
+/release
```
Comment (bcwaldon) [inline .gitignore-e]: This file is unnecessary, right?

Comment (ecnahc515) [inline .gitignore-e]: Whoops. Will fix.

**Comment by ecnahc515** [comment]
Fixed.

**Comment by bcwaldon** [comment]
LGTM

**Comment by bcwaldon** [comment]
@ecnahc515 merging for you so we can release this today

**Pull request merged**